addons/medical_treatment/XEH_preInit.sqf

```diff
@@ -34,4 +34,23 @@ GVAR(animDurations) = [] call CBA_fnc_createNamespace;
     ["AinvPknlMstpSnonWnonDnon_medic1", 10]
 ];
 
+// class names of medical facilities
+// global variable so it can be accessed by mission makers
+GVAR(facilityClasses) = [
```

Comment:
Wouldn't it make more sense to have the default list in config? So that mods like CUP/RHS can add compat?
Iterating over all vehicles is not an option though. So maybe a seperate config class for these? Maybe in CfgMagazineWells style to make adding onto it easy?

Reply:
We add CfgVehicles config entry and then compile the list at preStart? 
Or for your suggestion, would something like this work?
```cpp
class ACE_Medical_Facilities {
    BI[] = {
        "VanillaMedFac01_F",
        "VanillaMedFac02_F"
    };
    CUP[] = {
        "TK_GUE_WarfareBFieldhHospital_Base_EP1",
        "TK_GUE_WarfareBFieldhHospital_EP1"
    };
};
```

Reply:
I'd probably do a separate config like that, same as we're doing for the healing item replacement

Reply:
Yes that's what I thought about.

Not sure if we really need preStart. The problem with preStart is that we always have to add overhead of compileFinal to make it hacking safe.
preInit would be good enough here.
Or.. We could argue that it's not gameplay relevant enough to be worth hacking, and thus we don't care and don't need to make it compileFinal'ed?